packages/contracts-bedrock/src/libraries/Encoding.sol

```diff
@@ -133,4 +133,44 @@ library Encoding {
         }
         return (nonce, version);
     }
+
+    /// @notice Returns an appropriately encoded call to L1Block.setL1BlockValuesEcotone
+    /// @param basefeeScalar       L1 Basefee Scalar
+    /// @param blobBasefeeScalar   L1 Blob Basefee Scalar
```

Comment:
nit, the capitalization is inconsistent on these params vs. the ones below.

Reply:
fixed